restli-tools/src/test/extensions/validCase/BazExtensions.pdl

```diff
@@ -0,0 +1,10 @@
+/**
+ * Valid extension schema:
+ * The co-existence of @extension and @grpcExtension is allowed
+ */
+record BazExtensions includes Baz {
+  @extension.using = "finder: test"
+  @grpcExtension.rpc = "get"
```

Comment:
what would be the expectation for translating this annotation to proto during migration?

Reply:
The proto schema would look something like [this](https://jarvis.corp.linkedin.com/codesearch/result/?name=BarExtensions.proto&path=entity-relationship-infra%2Fgenerator%2Fsrc%2Ftest%2FprotoExtensions%2Fcom%2Flinkedin%2Fentityrelationship%2Fgenerator&reponame=linkedin-multiproduct%2Fentity-relationship-infra#50).